packages/fast-form-generator-react/src/form/form-item.checkbox.style.ts

```diff
@@ -0,0 +1,51 @@
+import { Direction, ellipsis, localizeSpacing, toPx } from "@microsoft/fast-jss-utilities";
+import { ComponentStyles, ICSSRules } from "@microsoft/fast-jss-manager";
+import { IFormItemCheckboxClassNameContract } from "../class-name-contracts/";
+import { applyLabelStyle } from "../utilities/form-input.shared-style.style";
+
+const styles: ComponentStyles<IFormItemCheckboxClassNameContract, {}> = {
+    formItemCheckbox: {
+        display: "flex",
+        flexDirection: "row"
+    },
+    formItemCheckbox_label: {
+        ...applyLabelStyle()
+    },
+    formItemCheckbox_input: {
+        position: "relative",
+        cursor: "pointer",
+        width: toPx(20),
+        height: toPx(20),
+        appearance: "none",
+        borderRadius: toPx(2),
+        boxShadow: "inset 0px 0px 4px rgba(0, 0, 0, 0.08)",
+        backgroundColor: "rgba(0, 0, 0, 0.04)",
+        "&:after, &:before": {
```

Comment:
you cannot have a before and after on an input, this needs to be on a span or other element because input elements do not have any content